Assets/Scripts/ImprovedCharSelection.cs

```diff
@@ -0,0 +1,124 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+// just to see the profile
+public class ImprovedCharSelection : MonoBehaviour
+{
+    private GameObject[,] characterList;
+    private int classIndex = 0;
+    private int colorIndex = -1;
+
+    private void Start()
+    {
+        classIndex = PlayerPrefs.GetInt("ClassSelected");
+        colorIndex = PlayerPrefs.GetInt("ColorSelected");
+
+        // presupun ca nu vor fi mai mult de 5 skinuri
+        // in principiu vom avea fix 3
+        characterList = new GameObject[transform.childCount, 3];
```

Comment:
I don't understand what this is. I believe you store here all the possible prefabs. Use English for comments.

Reply:
If this is what you are doing than could you refactor it? Or just comment it.